src/internal/pfsload/validator.go

```diff
@@ -138,7 +146,7 @@ func validate(client Client, commit *pfs.Commit, files []*file) (retErr error) {
 		}
 		files = files[1:]
 		return nil
-	})
+	}, true)
```

Comment:
Is this the don't use all the memory flag?

Reply:
Yeah, the file content is streamed when this is set.